### PR TITLE
[fix]: select,radio,fileselector, fileupload,textarea,modal

### PR DESCRIPTION
**Pull request by Shubham9455**

### CODE REVIEW

File: src/components/ui/textarea.tsx [hunk 1]
```diff
@@ -5,18 +5,19 @@ export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextArea
   label?: string;
   description?: string;
   error?: string;
+  width?: string;
   variant?: "default" | "filled"; // Added variant prop
 }
 
 const Textarea = React.forwardRef<HTMLTextAreaElement, TextareaProps>(
-  ({ className, label, description, error, variant = "default", ...props }, ref) => {
+  ({ className, label, description, error, variant = "default",width, ...props }, ref) => {
```
Comment (buskerone) [inline src/components/ui/textarea.tsx:13]: No need to do it now, I'll add eslint+prettier to fix lint issues like the space after the ",". 
About the width, do you think we should just use the className prop to keep using tailwind? I'm fine with passing it via style though.

File: src/components/ui/textarea.tsx [hunk 2]
```diff
@@ -5,18 +5,19 @@ export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextArea
...
     return (
-      <div className={cn("flex flex-col gap-1",className)}>
+      <div className={cn("flex flex-col gap-1",className)} style={{width}}>
         {label && <label className={`text-md  font-bold text-foreground mb-[${description ? '4px' : '12px'}]`}>{label}</label>}
         {description  && <p className={`text-sm text-muted-foreground mb-[12px]`}>{description}</p>}
```
Comment (buskerone) [inline src/components/ui/textarea.tsx:17]: I can see some extra spaces here.

File: src/components/ui/textarea.tsx [hunk 3]
```diff
@@ -5,18 +5,19 @@ export interface TextareaProps extends React.TextareaHTMLAttributes<HTMLTextArea
...
         <textarea
           className={cn(
-            "flex min-h-[80px] w-full rounded-md px-3 py-2 text-sm  text-foreground placeholder:text-muted-foreground focus-visible:outline-none focus-visible:border-2 focus-visible:border-foreground-actionable disabled:cursor-not-allowed disabled:bg-surface-muted",
+            "flex min-h-[80px] w-full rounded-md px-3 py-2 text-sm  text-foreground  text-md placeholder:text-muted-foreground placeholder:text-lg focus-visible:outline-none focus-visible:border-2 focus-visible:border-foreground-actionable disabled:cursor-not-allowed disabled:bg-surface-muted",
```
Comment (buskerone) [inline src/components/ui/textarea.tsx:20]: Extra spaces here as well: `text-sm  text-foreground  text-md`

**Review by buskerone — APPROVED** [review]
LGTM. Thanks for creating the stories. I put some comments regarding lint issues, nothing critical. We can add prettier+eslint later.

File: src/components/ui/textarea.tsx (hunk 1 above)
Comment (Shubham9455) [reply src/components/ui/textarea.tsx:13]: Okay, I’ll adjust the code to include the width within the class in the next PR.

**Pull request merged**